Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/BearerToken.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class BearerToken : HeaderBasedAuthorization
+    {
+        public override string headerPrefix { get; protected set; } = "Bearer ";
```

Comment:
Waarom is deze public?

Reply:
Na het lezen van de rest van de code, als we een public property of method introduceren "HeaderValue" in de HeaderBasedAuthorization dan hoeven we minder in de aanroeper te doen wat eigenlijk hierin hoort - en kunnen we die code ook versimpelen